test/byContract/IexecEscrow/IexecEscrowNative.ts

```diff
@@ -83,13 +83,34 @@ if (CONFIG.chains.default.asset === 'Native') {
                 await expect(iexecPocoAsAccountA.deposit(...depositArgs))
                     .to.changeEtherBalances(
                         [accountA, iexecPoco],
-                        [-depositNativetAmount, depositNativetAmount],
+                        [-depositNativeAmount, depositNativeAmount],
                     )
                     .to.changeTokenBalances(iexecPoco, [accountA], [depositAmount])
                     .to.emit(iexecPoco, 'Transfer')
                     .withArgs(AddressZero, accountA.address, depositAmount);
             });
 
+            it('Should deposit native tokens and return excess', async () => {
```

Comment:
Remainder is more explicit and explains that there is a division operation
```suggestion
            it('Should deposit native tokens and return remainder', async () => {
```